### DISCUSSION TITLE
Get RSSI from connected device?

### DISCUSSION THREAD

**Discussion by Giovanni-Fantin** [Navideck/universal_ble#68] [Q&A] [answered]
Hello everyone,

First of all: great library
My question is: is it possible to get the RSSI once connected to a device? or only during the scan process?

**Comment by rohitsangwan01** [comment, marked as answer]
@Giovanni-Fantin its not yet implemented in our plugin, Though its supported natively on [Apple](https://developer.apple.com/documentation/corebluetooth/cbperipheral/readrssi()?language=objc) and [Android](https://developer.android.com/reference/android/bluetooth/BluetoothGatt#readRemoteRssi()) only, 
Linux RSSI D-Bus property doesn't seem to be updated after connection. Windows and Web does not support this
